canonical-web-and-design/charmhub.io
# Move collections to store

## Done

- /collections is now /store - including the navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the navigation now says "Store"
 - clicking "Store" redirects to "/store" and shows the store page
